README.md

```diff
@@ -29,13 +29,27 @@ Will result in:
 **Lorem** ipsum dolor sit amet, consectetur adipiscing elit. ... Etiam bibendum **lorem** nec tempus sollicitudin. ... Sed in dapibus **lorem**. ... Nunc turpis ipsum, bibendum quis sodales sed, ullamcorper et **lorem**. Donec et metus hendrerit, interdum elit ut, dignissim dui.
 
 
+### For Laravel
+
+Add the source location to ``composer.json`` which adds the package to the composer autoload collection:
+```
+"autoload": {
+    "psr-4": {
+        "App\\": "app/", 
+        "Swis\\TextSnippet\\": "vendor/swisnl/textsnippet/src/"
+    }
+}
+```
+And run ``composer dump-autoload``
+
+
```

Comment:
This section can be removed as the change to your `composer.json` is not necessary. Composer knows what to do if the PSR-4 config is properly set in this package.